src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarPublisher.java

```diff
@@ -121,9 +132,14 @@ public CodeSonarPublisher(
             conditions = ListUtils.EMPTY_LIST;
         }
         this.conditions = conditions;
-
         this.credentialId = credentialId;
         this.visibilityFilter = visibilityFilter;
+        this.visibilityFilterNewWarnings = visibilityFilterNewWarnings;
+        this.codesonarProjectFile = codesonarProjectFile;
```

Comment:
I originally recommended that we name this "codesonarProjectFile" since I was under the impression that the "projectName" parameter was the Jenkins project name.  Now that I know better, I no longer feel that it is so important to include "codesonar" in this parameter name.

I think we should change this parameter to be named `projectFile`.  I think this will be more consistent with the codesonar commandline and it will be more consistent with the `projectName` parameter, which we are now treating as a CodeSonar project name (or path).